c-pallets/file-bank/src/lib.rs

```diff
@@ -304,7 +310,9 @@ pub mod pallet {
 						};
 						if now > info.deadline + frozen_day {
 							log::info!("clear user:#{}'s files", number);
+							// sub_purchased_space can return an Err but it is ignored?
 							let _ = T::MinerControl::sub_purchased_space(info.space);
```

Comment:
Added the processing of returned results, which will not end the method execution, but will output some logs:
https://github.com/CESSProject/cess/pull/88/files#diff-9c37b1a61a6e9394aac1e9383bf9d3fac9539378d1af8fcd75aed4f8a4f42967R312-R316